Expensify/App
# add settlement frequency link complete

- [ ] HOLD on https://github.com/Expensify/Web-Expensify/pull/33250



### Details

This PR needs this https://github.com/Expensify/Web-Expensify/pull/33250 to be merged before it works.

### Fixed Issues

$https://github.com/Expensify/Expensify/issues/199498

### Tests
1. Click on your profile picture
2. Go to your workspace
3. Go to issue cards
4. Click on Settlement Frequency
5. It should redirect you to olddot settings tab with this picture


<img width="1440" alt="Screenshot 2022-03-09 at 9 21 03 AM" src="https://user-images.githubusercontent.com/26106822/157441820-1f1b06a8-d018-48c3-ab0a-7502068e261b.png">



For example:
1. Click on the text input to bring it into focus
7. Upload an image via copy paste
8. Verify a modal appears displaying a preview of that image
--->


### PR Review Checklist

#### Contributor (PR Author) Checklist

#### PR Reviewer Checklist

### QA Steps



### Screenshots


#### Web


#### Mobile Web


#### Desktop


#### iOS


#### Android
